### PR TITLE
Data Science with Python : Convolutional Neural Networks

### LINKED ISSUE #362: Data Science with Python : Convolution Neural Networks
Welcome to 'DSWP' Team, good to see you here

This issue will helps readers in giving all the guidance that one needs to learn about Introduction to Convolution Neural Networks. Tutorial to Convolution Neural Networks and how it's applied using sample code.

To get assigned to this issue, add your serial numbers mentioned in the spreadsheet of "Data Science with Python", the approach one would follow and choice you prefer (Documentation, Audio, Video). You can go with all three or any number of options you're interested to work on.

If you had referred any resources, add them up in "DS Resources". Similarly if you had used datasets, include them in "DS Datasets".

Domain : Deep Learning

Mentors Assigned : Nithin / Gauri Bharadwaj

Points to Note :

The issues will be assigned on a first come first serve basis, 1 Issue == 1 PR.

"Issue Title" and "PR Title should be the same. Include issue number along with it.

Changes should be made inside the Datascience_With_Python/ directory & Datascience_With_Python branch.

Follow Contributing Guidelines & Code of Conduct before start Contributing.

This issue is only for 'GWOC' contributors of 'DSWP' domain.

All the best. Enjoy your open source journey ahead. 😎

### PR DESCRIPTION
**Pull request by samridhikapoor**
<hr>

## Description 📜

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. 
Fixes #362

<hr>

## Type of change 📝

- [ ] Au#dio (Should be in mp3 format Includes speech clarity, Concise ,Low distortion)
- [x] Video (Animations, screen-recordings, presentations and regular explanatory films are all possibilties etc)
- [ ] Doc#umentation (Content Creation in the form of codes or tutorials)
- [ ] Other (If you choose other, Please mention changes below) 

<hr>

## Domain of Contribution 📊

- [ ] Android Dev #(Flutter)
- [ ] Android Dev #(Java)
- [ ] Android Dev #(Kotlin)
- [ ] Backend Dev #(Java)
- [ ] Backend Dev #(.NET)
- [ ] Backend Dev #(PHP)
- [ ] Backend Dev #(Python)
- [ ] C/#CPP
- [ ] Competitive #Programming
- [ ] Cyber #Security
- [ ] DS#A
- [ ] Data#base
- [x] Datascience with Python
- [ ] Datascience with #R
- [ ] Frontend Dev #HTML/CSS/JS
- [ ] Frontend Dev #React/Angular/Vue
- [ ] Go#lang
- [ ] Interview #Prep
- [ ] Java_#Domain
- [ ] Java#Script
- [ ] ME#RN
- [ ] Machine #Learning
- [ ] Open #Source
- [ ] Python_#Domain
- [ ] Ru#st
- [ ] Statis#tics
- [ ] UI/#UX

<hr>

## Checklist ✅

- [x] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] My changes generates no new warnings.
- [x] I'm GWOC'21 contributor

<hr>

## Screenshots / Gif (Optional) 📸

<hr>

### CODE REVIEW

File: Datascience_With_Python/Deep Learning/Tutorials/Data Science with Python  Convolution Neural Networks.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+--------------------------------------
+### What will we gather ?
+- How a computer reads an image
+- Why don't we prefer to use fully connected networks for image recognition
+- What is convolutional neural network
+- How does convolutional neural network work
+- A use case of Convolutional neural network
+- Summary
+
+[![Convolutional neural networks](https://www.dropbox.com/s/wb387q7kil6z0uy/Capture.PNG?dl=0&raw=1)](https://drive.google.com/file/d/10fw-kCYAauCNG20OAA2DUqHk1rUE9GuM/view "Cnvolutional Neural Networks")
```
Comment (Pushpit07) [inline Datascience_With_Python/Deep Learning/Tutorials/Data Science with Python  Convolution Neural Networks.md:11]: Please correct the typo in `Convolutional`

**Review by prathimacode-hub — CHANGES_REQUESTED** [review]
Kindly create a project folder of Convolutional Neural Networks under "/Datascience_with_Python/Deep Learning/Videos/" repo
Add README.md file with more details on the topic

Kindly work on it. @samridhikapoor

**Comment by samridhikapoor** [comment]
Okay Ma'am I am working on it.

**Comment by samridhikapoor** [comment]
@prathimacode-hub I have made the desired changes. Kindly review the same
Thanks

**Review by prathimacode-hub — CHANGES_REQUESTED** [review]
Add advantages, applications and conclusion of the topic at the end.
And also file name, kindly change it to README.md @samridhikapoor

**Comment by samridhikapoor** [comment]
> Add advantages, applications and conclusion of the topic at the end. And also file name, kindly change it to README.md @samridhikapoor

I have made the desired changes. Kindly review the same
Thanks

**Comment by prathimacode-hub** [comment]
All changes not done. @samridhikapoor

**Comment by samridhikapoor** [comment]
> All changes not done. @samridhikapoor

Ma'am, Please check it now.

**Review by prathimacode-hub — APPROVED** [review]
PR approved. Great efforts✨🙌@samridhikapoor

**Pull request merged**